crates/bevy_pbr/src/light_probe/environment_filter.wgsl

```diff
@@ -0,0 +1,396 @@
+#import bevy_render::maths::{PI, PI_2};
+#import bevy_pbr::utils::{rand_f, build_orthonormal_basis};
+
+struct FilteringConstants {
+    mip_level: f32,
+    sample_count: u32,
+    roughness: f32,
+    blue_noise_size: vec2f,
+}
+
+@group(0) @binding(0) var input_texture: texture_2d_array<f32>;
+@group(0) @binding(1) var input_sampler: sampler;
+@group(0) @binding(2) var output_texture: texture_storage_2d_array<rgba16float, write>;
+@group(0) @binding(3) var<uniform> constants: FilteringConstants;
+@group(0) @binding(4) var blue_noise_texture: texture_2d<f32>;
+
+// from bevy_anti_aliasing/src/taa/taa.wgsl
```

Comment:
Tbh might be good to put this in bevy_render somwhere, if you feel like it. I may want to use it for Solari too, haven't tried it yet.

Reply:
resolved by https://github.com/mate-h/bevy/pull/8